### DISCUSSION TITLE
Git Exporter

### DISCUSSION THREAD

**Discussion by stephenyhb** [Poeschl/Hassio-Addons#263] [Q&A]
Why I keep this error ?

[master 2fc91ee] Home Assistant Git Exporter
 3 files changed, 1 insertion(+), 1 deletion(-)
remote: Invalid username or password.
fatal: Authentication failed for 'https://github.com/stephenyhb/MyHA.git/'

**Comment by Poeschl** [comment]
Are your GitHub credentials correct? Have you tried to remove and add the addon again?
